configs/repos/jedi/packages/faux/package.py

```diff
@@ -0,0 +1,39 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Faux(CMakePackage):
+    """Faux is a collection of selected Fortran utility libraries, extracted from the IFS."""
+
+    # DH* TODO UPDATE
+    homepage = "https://github.com/JCSDA-internal/faux"
+    git = "https://github.com/JCSDA-internal/faux.git"
+
+    maintainers = ['climbfuji']
+
+    version('develop', branch='develop', no_cache=True, preferred=True)
+    #version('bugfix-compile-failures', branch='bugfix/fix_compile_failures', no_cache=True)
+
+    variant('enable_mpi',   default=True, description='Use MPI?')
```

Comment:
Just a general comment that the preferred variant names are `+mpi`, `+openmp`, `+fckit`. 

https://spack.readthedocs.io/en/latest/packaging_guide.html#variant-names

Reply:
No need for any special logic with a CMakePackage. It's built into Spack.

```
self.define_from_variant('BUILD_WITH_FLAP', 'flap'),
self.define_from_variant('BUILD_WITH_PFLOGGER', 'pflogger'),
self.define_from_variant('ESMA_USE_GFE_NAMESPACE', 'esma_gfe_namespace'),
self.define_from_variant('BUILD_SHARED_MAPL', 'shared'),
```

Reply:
That's great. I'll make those changes in the afternoon and retest.

Reply:
Done.